src/ansys/fluent/core/services/settings.py

```diff
@@ -263,7 +263,12 @@ def _extract_static_info(self, info: SettingsModule.StaticInfo) -> dict[str, Any
             ret["object-type"] = self._extract_static_info(info.object_type)
         if info.help:
             ret["help"] = info.help
-
+        try:
```

Comment:
wondering why/when this will throw

Reply:
@mkundu1 @seanpearsonuk 

If we use `if info.HasField("attrs"):` we get following error.

![image](https://github.com/ansys/pyfluent/assets/106588300/1d35e5e9-1844-4bf9-881d-ec5380ae386e)

Reply:
Good point, `HasField` is not supported for repeated/map fields. Perhaps in that case we can remove the if condition altogether as `info.attrs` should always exists as map, IIUC.

Reply:
Yes, verified. Following change works fine.

![image](https://github.com/ansys/pyfluent/assets/106588300/21019ea6-90a8-4f40-986c-2f7472560d21)

Thank you.